parser.js

```diff
@@ -32,6 +32,10 @@ exports.parseForESLint = function parseForESLint(code, options) {
             options = Object.assign({}, options, { jsx: tsx });
         }
     }
+    if (options.ecmaFeatures && typeof options.ecmaFeatures.jsx === "boolean") {
```

Comment:
can you check if `options.jsx` is not defined already here?

we don't want to change behavior for ppl who have some options already set.